Cargo.lock

```diff
@@ -4719,7 +4719,7 @@ dependencies = [
 
 [[package]]
 name = "tower-batch-control"
-version = "0.2.40"
+version = "0.2.41-beta.1"
```

Comment:
Fine with this even if 0.x.y is considered initial development release(s)

Reply:
Technically 0.x.y are initial development releases, in practice tower is running in production and on 0.4.13.